bayesian/api_v1.py

```diff
@@ -737,9 +738,23 @@ def post():
         github_url = request.form.get("github_url")
         ref = request.form.get('github_ref')
         user_email = request.headers.get('UserEmail')
+        scan_repo_url = request.headers.get('scanRepoUrl')
+        print('Email ' + user_email)
+        print('GitUrl ' + scan_repo_url)
         if not user_email:
             user_email = decoded.get('email', 'bayesian@redhat.com')
 
+        if scan_repo_url is not None and scan_repo_url is not "":
+            try:
+                api_url = GEMINI_SERVER_URL
+                data = {'git-url': scan_repo_url,
+                        'email-ids': [user_email]}
```

Comment:
do we want to send a list here. I guess there will always be a single email associated to the build?

Reply:
@samuzzal-choudhury wants its in a list format